docs/coordinates/matchsep.rst

```diff
@@ -80,8 +78,8 @@ astronomy convention (positive angles East of North)::
     >>> c1.position_angle(c2).to(u.deg)  # doctest: +FLOAT_CMP
     <Angle 44.97818294 deg>
 
-The combination of :meth:`~astropy.coordinates.SkyCoord.separation` and
-:meth:`~astropy.coordinates.SkyCoord.position_angle` thus give a set of
+The combination of :meth:`~astropy.coordinates.BaseCoordinateFrame.separation`
+and :meth:`~astropy.coordinates.SkyCoord.position_angle` thus give a set of
```

Comment:
For another time, but it really is not very logical for the baseframe to provide `separation` but not `position_angle` - they're very similar pieces of code...

Reply:
@eerovaher , was this applied?

Reply:
Or is this not applicable?

Reply:
We should look into moving the `position_angle()` method from `SkyCoord` to `BaseCoordinateFrame`, but I can't think of a good reason why we'd want to do it in this pull request. Removing the `SkyCoord` separation methods is a meaningful and a reasonable large change on its own and changing the implementation of `position_angle()` can be done independently.

Reply:
Definitely was meant for another PR!